### DISCUSSION TITLE
chat() doesn't seem to be able to copy parts of it, only the whole thing 🤔?

### DISCUSSION THREAD

**Discussion by alkene5** [johnlindquist/kit#1398] [Q&A] [answered]
<img width="760" alt="image" src="https://github.com/johnlindquist/kit/assets/61105068/cca25fff-e86c-4324-ace3-5a44ca060d88">

**Comment by johnlindquist** [comment, marked as answer]
@alkene0005 This will be fixed in the next build

**Reply by alkene5** [reply]
The current situation is that only the right-click menu can be copied successfully, and the shortcut key(command + c) cannot be copied
